packages/material-ui/src/InputBase/InputBase.js

```diff
@@ -716,7 +716,7 @@ InputBase.propTypes /* remove-proptypes */ = {
   /**
    * The size of the component.
    */
-  size: PropTypes.oneOf(['medium', 'small']),
+  size: PropTypes.oneOf(['large', 'medium', 'small']),
```

Comment:
It should be https://github.com/mui-org/material-ui/blob/398684450a89ef2cf167d56be3e1a95ea1763f7a/packages/material-ui/src/TextField/TextField.js#L382-L385

Reply:
Thanks for the help @oliviertassinari